test/docker-tests/src/test/java/it/unibz/inf/ontop/docker/ParallelScenarioManifestTestUtils.java

```diff
@@ -63,7 +64,7 @@ public static TestSuite suite(ParallelScenarioTest.ParallelFactory factory) thro
 		if ("jar".equals(url.getProtocol())) {
 			// Extract manifest files to a temporary directory
 			try {
-				tmpDir = FileUtil.createTempDir("scenario-evaluation");
```

Comment:
Method removed in RDF4J 4 so replaced with a more standard approach that doesn't depend on any dependency's version.